test/rekt/README.md

```diff
@@ -0,0 +1,16 @@
+# Reconciler Tests
+
+This is the staging location for the new
+[e2e testing framework](https://github.com/knative-sandbox/reconciler-test).
+
+To run the tests on an existing cluster:
+
+```bash
+SYSTEM_NAMESPACE=knative-eventing go test -count=1 -v -tags=e2e ./test/rekt/...
+```
+
+To run just one test:
+
+```bash
+SYSTEM_NAMESPACE=knative-eventing go test -count=1 -v -tags=e2e -run TestKafkaSource ./test/rekt/...
```

Comment:
I think we need a bit more here, but we can add that in a later PR

e.g. a proper strimzi needs to be around, like the setup does for the CI on the repo.

I also got:

```
2021/05/03 17:27:55 Published docker.io/matzew/eventshub@sha256:4a6dd402d1cf0592e89825ed733cb4262a558cef104d68fcd36955b9d76f12d8
    kafka_source_test.go:54: could not copy secret(strimzi-tls-secret): secrets "strimzi-tls-secret" not found
--- FAIL: TestKafkaSource (16.95s)

```

due to missing setup instructions - but that's NOT a blocker for this PR

Reply:
I'll open an issue to keep track of it